akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/RoutersSpec.scala

```diff
@@ -115,6 +115,41 @@ class RoutersSpec extends ScalaTestWithActorTestKit("""
       }
     }
 
+    "supports broadcast" in {
+      trait Cmd
+      case object ReplyWithAck extends Cmd
+      case class BCast(cmd: Cmd) extends Cmd
+
+      val atomicInt = new AtomicInteger(0)
```

Comment:
A better alternative to using an atomic counter would be to send `ctx.self.path` to the probe when receiving the broadcast and verify you got 4 different paths (set). That way you would really know each routee got one message (with this all 4 could have gone to the same and test would pass)

Reply:
good point